contracts/contracts/harvest/Harvester.sol

```diff
@@ -0,0 +1,388 @@
+     * @param _swapToken Address of the token to swap.
+     */
+    function _swap(address _swapToken)
+        internal
+        returns (uint256[] memory swapResult)
+    {
+        RewardTokenConfig storage tokenConfig = rewardTokenConfigs[_swapToken];
+        require(
+            tokenConfig.uniswapV2CompatibleAddr != address(0),
+            "Swap token is missing token configuration."
+        );
+
+        address priceProvider = IVault(vaultAddress).priceProvider();
+        address[] memory allAssets = IVault(vaultAddress).getAllAssets();
+
+        if (tokenConfig.uniswapV2CompatibleAddr != address(0)) {
+            IERC20 swapToken = IERC20(_swapToken);
+            uint256 balance = swapToken.balanceOf(address(this));
+            if (balance > 0) {
+                uint256 maxBalanceToSwap = balance;
+                if (tokenConfig.liquidationLimit != 0) {
+                    maxBalanceToSwap = Math.min(
+                        balance,
+                        tokenConfig.liquidationLimit
+                    );
+                }
+
+                // This'll revert if there is no price feed
+                uint256 oraclePrice = IOracle(priceProvider).price(_swapToken);
+                // Oracle price is 1e8, USDT output is 1e6
+                uint256 minExpected = (maxBalanceToSwap *
+                    oraclePrice *
+                    (1e4 - tokenConfig.allowedSlippageBps)).scaleBy( // max allowed slippage
+                    6,
+                    Helpers.getDecimals(_swapToken) + 8
+                ) / 1e4; // fix the max slippage decimal position
+
+                // Uniswap redemption path
+                address[] memory path = new address[](3);
+                path[0] = _swapToken;
+                path[1] = IUniswapV2Router(tokenConfig.uniswapV2CompatibleAddr)
+                    .WETH();
+                path[2] = allAssets[1]; // USDT
+
+                swapResult = IUniswapV2Router(
+                    tokenConfig.uniswapV2CompatibleAddr
+                ).swapExactTokensForTokens(
+                        maxBalanceToSwap,
+                        minExpected,
+                        path,
+                        address(this),
+                        block.timestamp
+                    );
+
+                IERC20 usdt = IERC20(allAssets[1]); // USDT
+                uint256 usdtBalance = usdt.balanceOf(address(this));
+                uint32 vaultBps = 1e4 - tokenConfig.harvestRewardBps;
+                require(
+                    tokenConfig.harvestRewardBps > 0,
+                    "Harvest rewards can not be zero"
+                );
+                require(
+                    vaultBps > tokenConfig.harvestRewardBps,
+                    "Address calling harvest is receiving more rewards than the vault"
+                );
+
+                // When governor calls the swap function send full reward amount to the Vault
+                if (isGovernor()) {
```

Comment:
@DanielVF @tomlinton made this change. The idea is that governor calling `harvest` and `swap` and `harvestAndSwap` doesn't get the 1% cut of the sold rewards. Let me know if any objections. 

Also would it be gas cheaper to pass down to this `_swap` function whether to send 1% to the `msg.sender` instead of calling `isGovernor ()`?